rioxarray/raster_dataset.py

```diff
@@ -147,7 +149,8 @@ def reproject_match(self, match_data_array, resampling=Resampling.nearest):
             Dataset with the target resolution and projection.
         resampling: rasterio.enums.Resampling, optional
             See :func:`rasterio.warp.reproject` for more details.
-
+        **reproject_kwargs:
+            Other options to pass to :obj:`rioxarray.raster_array.reproject()`
```

Comment:
```suggestion
            Other options to pass to :meth:`rioxarray.raster_array.RasterArray.reproject`
```

Reply:
Ah nice, just commented above on not knowing how the docs quite went. Thanks!